scripts/dts/gen_defines.py

```diff
@@ -868,7 +871,7 @@ def write_global_macros(edt):
     compat2buses = defaultdict(list)  # just for "okay" nodes
     for compat, okay_nodes in edt.compat2okay.items():
         for node in okay_nodes:
-            bus = node.on_bus
+            bus = node.on_buses
```

Comment:
But this is a list now; don't you need to use `.extend` below instead of `.append`, or something like that, along with any other needed changes? This doesn't seem right; what am I missing?

Reply:
Fixed.